### ISSUE TITLE
ui5-table keyboard navigation does not work

### ISSUE THREAD

**Issue by pskelin** [SAP/ui5-webcomponents#861]
**Describe the bug**
the table rows cannot be focused, hence keyboard navigation doesn't work at all.

**To reproduce**
1. Go to https://sap.github.io/ui5-webcomponents/test-resources/sap/ui/webcomponents/main/pages/Table.html
2. Use tab key
3. table row never receives focus

**Expected behavior**
table rows should receive focus and keyboard navigation via arrow keys should work

**Context**
- UI5 Web Components version: 1.0.0-rc.4

**Affected components** *(if known)*
ui5-table, ui5-table-row
